### ISSUE TITLE
Tests fail

### ISSUE THREAD

**Issue by emoszkowski** [matthieugomez/Gensys.jl#4]
The test for `gensysct` is failing. Is that expected behavior or is something going wrong on my end?

Also, it might be better practice to either test on a smaller system or to move the giant arrays into an HDF5/JLD file, rather than to have large, unwieldy arrays hard-coded into the tests.

**Comment by emoszkowski** [comment]
To clarify, this is on master before opening the `gensys-type` branch.

**Comment by sglyon** [comment]
Do the tests pass on the gensys-type branch?

## EDIT

Also, I agree with your comment on moving test data out of the source files. Because these are just numerical arrays, I can't decide if it would be best to add a heavy dependency like HDF5/JLD, or if we could get away with just reading in a csv/JSON/YAML version of each of the matrices. 

I think JSON.jl is a much lighter dependency than HDF5/JLD

**Comment by emoszkowski** [comment]
The tests don't pass on `gensys-type` either, but they fail the same way as they do on master. I'm happy to convert the data to JSON or csv format there or on a separate branch, but I'm worried about writing tests to pass rather than doing a rigorous test. I might try with the DSGE.jl test data directly and see how that works.

**Comment by sglyon** [comment]
That's a really good point -- we don't want to just get tests that pass. 

Do you have good tests for the discrete time functionality in DSGE.jl?

**Comment by emoszkowski** [comment]
Yup, we do. We got it by making sure we get the same results as Chris Sims's MATLAB code.

**Comment by JulienPascal** [comment]
Hi,

Thanks for the great work. 

I have also noticed that tests fail on julia v0.6:
```julia
Test Failed
  Expression: sumabs2(G1 .- G1_solution) <= 1.0e-5
   Evaluated: 0.0034548960384160276 <= 1.0e-5
ERROR: LoadError: There was an error during testing
while loading /home/julien/.julia/v0.6/Gensys/test/runtests.jl, in expression starting on line 497
```

Same question as above: is that the expected behavior ?
